jvm-packages/xgboost4j/src/main/java/ml/dmlc/xgboost4j/java/Tensor.java

```diff
@@ -0,0 +1,156 @@
+/*
+ Copyright (c) 2021 by Contributors
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+package ml.dmlc.xgboost4j.java;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * A Tensor to hold the prediction result.
+ */
+public class Tensor {
```

Comment:
Could this class have a direct accessor for each element which does the appropriate indexing? And some documentation which states it's row-major, as otherwise the `getRawResult` function isn't really documented.